lain_sdk/yaml/conf.py

```diff
@@ -12,4 +12,4 @@
 
 PRIVATE_REGISTRY = None if etc is None else etc.get('private_docker_registry', None)
 
-DOMAIN = None if etc is None else etc.get('DOMAIN', 'lain.local')
+DOMAIN = None if etc is None else etc.get('domain', 'lain.local')
```

Comment:
I think there is no need to change the name, at least ensure that the old usage is compatible。